recipes/vaex/meta.yaml

```diff
@@ -0,0 +1,103 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "vaex" %}
+{% set version = "1.0.0b2" %}
```

Comment:
There is an old one, but it's better to wait till the 1.0.0 release. I wanted to see how this all works, is that ok? 
Btw, how to treat dependencies that I'm not the author of, shall I write the recipes myself and make a PR on a different branch?